### ISSUE TITLE
Can't create new stuff

### ISSUE THREAD

**Issue by clockworkerr** [Cheeteau/PPG-Mod-Creator#3]
So, I was trying to create an object in the PPG mod creator, but it didn't allow me because of some error.
![image](https://user-images.githubusercontent.com/103432606/162750621-2978ce6b-7cc7-4b4c-a471-61afa9e57934.png)
I do not know how to reproduce it, but it happened when my internet was being stupid and i was creating a lot of stuff.
I've tried everything: restarting the computer, closing and re-opening the website but it doesn't work.

**Comment by Cheeteau** [comment]
Hey, it seems like the website considers you already created a mod element with this name. Try to see if it already exists, and if not it means you will have to delete your brother cache which also means the loss of all of your work.

**Comment by clockworkerr** [comment]
Oh. well is there anything else i can do?

**Comment by clockworkerr** [comment]
Ok, its a meme mod for friends you won't understand much
![image](https://user-images.githubusercontent.com/103432606/162963306-400bae85-8751-4fb6-a03f-351e9d8fb600.png)
![image](https://user-images.githubusercontent.com/103432606/162963381-e02d3f1e-bcac-429e-bd69-6d792d8119ab.png)
![image](https://user-images.githubusercontent.com/103432606/162963407-85e2afad-8170-46a3-bc7e-a6ae5f173199.png)

**Comment by Cheeteau** [comment]
Mhm, is it happening when you use another name for the item you wanna create?

**Comment by clockworkerr** [comment]
Yes, on anything i create it doesn't show up in the explorer. And when i create it with the same name, it says it already exists.

**Comment by Cheeteau** [comment]
Okay. Try to open the developer tools by pressing Ctrl+Shift+I or f12 if you're on chrome.
Once you're on it, go on "Application",
![image](https://user-images.githubusercontent.com/29103374/162964976-9b7dd173-3481-4a9b-8dfe-67d98871aabe.png)
and then go in "Local Storage" > "https://cheeteau.github.io".
![image](https://user-images.githubusercontent.com/29103374/162965084-848d5fa4-b040-47d1-b7b1-0d2879420d90.png)

Once you're on it, select "items", and copy the value.
![image](https://user-images.githubusercontent.com/29103374/162965188-bc35470b-f8a1-4665-8f41-8239d2afef2b.png)

Past the value in a new txt file, and send it to me so I can check it out.

**Comment by clockworkerr** [comment]
Ok, one sec

**Comment by clockworkerr** [comment]
[{category: "Entities", data: {type: "Human", audio: null,…}},…]
0: {category: "Entities", data: {type: "Human", audio: null,…}}
1: {category: "Entities", data: {type: "Android", audio: null,…}}
2: {category: "Entities", data: {type: "Gorse", audio: null,…}}
3: {category: "Entities", data: {type: "Android", audio: null,…}}
4: {category: "Entities", data: {type: "Human", audio: null,…}}
5: {category: "Entities", data: {type: "Human", audio: null,…}}
6: {category: "Entities", data: {type: "Human", audio: null,…}}
7: {category: "Entities", data: {type: "Human", audio: null,…}}
8: {category: "Entities", data: {type: "Human", audio: null,…}}
9: {category: "Misc.", data: {type: "Object", audio: null,…}}
10: {category: "Entities", data: {type: "Android", audio: null,…}}
11: {category: "Entities", data: {type: "Human", audio: null,…}}
12: {category: "Explosives", data: {type: "General Purpose Bomb", audio: null,…}}
13: {category: "Entities", data: {type: "Human", audio: null,…}}
14: {category: "Entities", data: {type: "Human", audio: null,…}}
15: {category: "Entities", data: {type: "Human", audio: null,…}}
16: {category: "Firearms", data: {type: "Blaster Rifle",…}}
17: {category: "Entities", data: {type: "Human", audio: null,…}}
18: {category: "Entities", data: {type: "Human", audio: null,…}}
19: {category: "Entities", data: {type: "Human", audio: null,…}}
20: {category: "Entities", data: {type: "Human", audio: null,…}}
21: {category: "Entities", data: {type: "Human", audio: null,…}}
22: {category: "Entities", data: {type: "Human", audio: null,…}}
23: {category: "Entities", data: {type: "Human", audio: null,…}}
24: {category: "Entities", data: {type: "Android", audio: null,…}}
25: {category: "Firearms", data: {type: "Grenade Launcher",…}}
26: {category: "Firearms", data: {type: "Sniper Rifle",…}}
27: {category: "Misc.", data: {type: "Object", audio: null,…}}
28: {category: "Entities", data: {type: "Human", audio: null,…}}
29: {category: "Misc.", data: {type: "Object", audio: null,…}}
30: {category: "Entities", data: {type: "Human", audio: null,…}}
31: {category: "Misc.", data: {type: "Object",…}}
32: {category: "Entities", data: {type: "Android", audio: null,…}}
33: {category: "Entities", data: {type: "Human", audio: null,…}}
34: {category: "Entities", data: {type: "Human", audio: null,…}}
35: {category: "Entities", data: {type: "Human", audio: null,…}}
36: {category: "Explosives", data: {type: "Red Barrel", audio: null,…}}

**Comment by Cheeteau** [comment]
Could you past those in a new text file? Like on notepad or something? So I could get them into my own browser.

**Comment by clockworkerr** [comment]
[ppgmc.txt](https://github.com/Cheeteau/PPG-Mod-Creator/files/8473032/ppgmc.txt)

**Comment by Cheeteau** [comment]
You sure you copied them right? Normally it's supposed to be a lot bigger

**Comment by clockworkerr** [comment]
I think so. let me double check

**Comment by Cheeteau** [comment]
Oh, I know how you copied them.
Do not copy this from here:
![image](https://user-images.githubusercontent.com/29103374/162967338-5789ca7e-4022-413b-827f-af3f0a34067e.png)

What you need to do is double click on here:
![image](https://user-images.githubusercontent.com/29103374/162967403-a08949fc-7e4d-449c-8406-fd6502105ab4.png)
Press CTRL+C, and past it in your txt file.
It should look more like that:
![image](https://user-images.githubusercontent.com/29103374/162967571-d98d8a11-8c81-4f5e-905f-e155f1f7ac70.png)

**Comment by clockworkerr** [comment]
Ohh ok

**Comment by clockworkerr** [comment]
My browser and notepad kept crashing, but i've managed to do i
[ppgmc.txt](https://github.com/Cheeteau/PPG-Mod-Creator/files/8473100/ppgmc.txt)
t

**Comment by Cheeteau** [comment]
Hi again, I found out what was going on. It seems like you created too much items. It seems like the storage of the browsers have a quota, which means there's a limited amount of data you can keep in it. It seems like I didn't think about it, so I think I will either set a limited amount of items you can create (which is probably the best to do), or store everything somewhere else which is more risky cause if people's pc aren't strong enough, it can lead to a crash.

**Comment by clockworkerr** [comment]
Well you could maybe make an option for it, i think if thats not too hard
or you could make it so once you reach the maximum amount, the other items get moved to somewhere else if you get what i mean

**Comment by Cheeteau** [comment]
I could try. It's not too hard to do but maybe a bit risky, as it wouldn't be optimized at all for low end pcs.
